perses/tests/test_repex.py

```diff
@@ -53,17 +55,18 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
 
             # Set up repex simulation
             reporter_file = os.path.join(temp_dir, f"{wt_name}-{mutant_name}.nc")
-            reporter = MultiStateReporter(reporter_file, checkpoint_interval=10)
+
+            reporter = MultiStateReporter(reporter_file, checkpoint_interval=100)
             hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=4.0 * unit.femtoseconds,
-                                                                                   collision_rate=1.0 / unit.picosecond,
-                                                                                   n_steps=50,
-                                                                                   reassign_velocities=False,
-                                                                                   n_restart_attempts=20,
-                                                                                   splitting="V R R R O R R R V",
-                                                                                   constraint_tolerance=1e-06),
-                                                                                   replica_mixing_scheme='swap-all',
-                                    hybrid_factory=htf,
-                                    online_analysis_interval=None)
+                                                                          collision_rate=1.0 / unit.picosecond,
+                                                                          n_steps=250,
```

Comment:
Would this still work if we reduced `n_steps` to 125?

Reply:
Changes in https://github.com/choderalab/perses/pull/1065/commits/693f41c45984fa35364005021c7581f46a620382